excludes/id.php

```diff
@@ -107,17 +116,19 @@
     'Rwanda',
     'Saint Barthelemy',
     'Saint Helena',
-    'Saint Kitts And Nevis',
+    'Saint Kitts and Nevis',
```

Comment:
The keys to exclude are specified here.

`'Saint Kitts And Nevis' !== 'Saint Kitts and Nevis'`

See: https://github.com/Laravel-Lang/lang/blob/master/source/en.json#L441